fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1163,6 +1164,57 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'reminders - create and delete',
```

Comment:
Do you think it might be worthwhile to have an additional test that has multiple accounts with reminders?